versions/spack.ver

```diff
@@ -0,0 +1,22 @@
+export spack_stack_ver=1.6.0
+export spack_env=gsi-addon
+
+export cmake_ver=3.23.1
+
+export jasper_ver=2.0.32
+export libpng_ver=1.6.37
+export zlib_ver=1.2.13
+
+export hdf5_ver=1.14.0
+export netcdf_c_ver=4.9.2
+export netcdf_fortran_ver=4.6.1
+
+export bacio_ver=2.4.1
+export nemsio_ver=2.5.4
+export sp_ver=2.5.0
+export sigio_ver=2.3.2
+export w3emc_ver=2.10.0
+export bufr_ver=11.7.0
+
+export grib_util_ver=1.3.0
```

Comment:
Some of these libraries are not loaded.  There's likely no harm in including them, but suggest paring this down to only the loaded modules:

```suggestion
export hdf5_ver=1.14.0
export netcdf_c_ver=4.9.2
export netcdf_fortran_ver=4.6.1

export bacio_ver=2.4.1
export nemsio_ver=2.5.4
export sp_ver=2.5.0
export sigio_ver=2.3.2
export w3emc_ver=2.10.0
export bufr_ver=11.7.0

```

Reply:
It looks like spack.ver is not used for wcoss2, right?

Reply:
Correct. It is not used _yet_. Once we have the newer spack-stack installed on WCOSS2 we won't use `spack.ver` on WCOSS2.